gef-ui/server/controllers/amendments/helpers/map-submitted-to-checker-email-variables.ts

```diff
@@ -59,8 +61,15 @@ const mapSubmittedToCheckerEmailVariables = ({
     formattedFacilityValue = `${currencySymbol}${value}`;
   }
 
-  const recipientName = `${user.firstname} ${user.surname}`;
-  const sendToEmailAddress = user.email;
+  const makersName = `${user.firstname} ${user.surname}`;
+  const makersEmail = user.email;
+
+  const checkersName = `${checker.firstname} ${checker.surname}`;
+  const checkersEmail = checker.email;
+
+  const dateSubmittedByMaker = format(new Date(), DATE_FORMATS.DD_MMMM_YYYY);
```

Comment:
```suggestion
  const dateSubmittedByMaker = format(now(), DATE_FORMATS.DD_MMMM_YYYY);
```

Function available in libs/common.

Reply:
done